{{ cookiecutter.target }}/etc/zope.conf

```diff
@@ -11,14 +11,10 @@ verbose-security {{ "on" if cookiecutter.verbose_security =="True" else "off" }}
 
 default-zpublisher-encoding utf-8
 
-{%- if cookiecutter.environment %}
 <environment>
-{%- for key, value in cookiecutter.environment|dictsort %}
-    {{ key }} {{ value }}
-{%- endfor %}
-
+    CHAMELEON_CACHE {{ cookiecutter.environment_CHAMELEON_CACHE | abspath }}
+    zope_i18n_compile_mo_files {{ cookiecutter.environment_zope_i18n_compile_mo_files }}
```

Comment:
Here a feature is lost, adding arbitrary environment strings. So we should loop over all strings starting with `environment_`, right/